docs/src/modules/components/overview/charts/featuresHighlight/FeaturesHighlight.tsx

```diff
@@ -25,11 +25,10 @@ export default function FeaturesHighlight() {
         Using MUI X Charts
       </Typography>
       <Typography variant="h4" sx={{ textAlign: 'center', fontWeight: 600 }}>
-        Visualize your data with effortless elegance
+        Elegant, effortless data visualization
```

Comment:
Should this be `Elegant and effortless` to reinforce both?

Reply:
Yeah, the meaning is always the same. I just think `and` in this case reinforces the idea that they are both true to the same extent in regards to "data visualisation". 

Could be preference, so not a big deal though 😆